lib/common/collections/wp_items/detectable.rb

```diff
@@ -31,7 +31,7 @@ def aggressive_detection(wp_target, options = {})
         request.on_complete do |response|
           progress_bar.progress += 1 if options[:show_progression]
 
-          if target_item.exists?(exist_options, response)
+          if target_item.exists?(exist_options, response) or options[:display_name]

```

Comment:
Bad things will happen due to that, i.e with ```-e vp,ud```, all known vulnerable plugins will be reported as present on the blog